smolmodels/directives.py

```diff
@@ -0,0 +1,65 @@
+# smolmodels.directives.py
+"""
+This module defines the `Directive` class, which represents instructions that guide the model
```

Comment:
```suggestion
This module defines the `Directive` class, which represents technical instructions that guide the model
```
nitpick: trying to make it clear that a directive is defining technical specifications for model building

Reply:
Good suggestion, commiting it 🔥